src/components/molecules/Header/Header.tsx

```diff
@@ -51,11 +54,7 @@ export function NHSNotifyHeader({ className, dataTestId }: HeaderType) {
           id='content-header'
         >
           {/* I am currently testing the link wrapper, this will change later when we implement auth as the link will change based on auth state */}
-          <div className='nhsuk-account__login' data-testid='login-link'>
-            <Link className='nhsuk-account__login--link' href='/'>
-              {content.components.headerComponent.links.logIn}
-            </Link>
-          </div>
+          {pathname?.includes('auth') ? '' : <LoginStatus />}
```

Comment:
Can we at least have a const somewhere for this?

Reply:
i don't really understand what this is asking for!

Reply:
A named `const` for the data `auth`, e.g. `const LOGIN_PAGE_PATH = 'auth'`.  Helps the reader understand *why* it does what it does.